packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -120,9 +121,13 @@ const AccountTransferForm = observer(
             resetConverter,
         } = crypto_fiat_converter;
 
+        const is_migration_status_present =
```

Comment:
in files like this one, this check can be done with local state. so it would be better if we keep things local and do not bring in logic from shared file.